<ui version="4.0" >
 <class>SimulParams</class>
 <widget class="QDialog" name="SimulParams" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parametros de Simulacion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>&amp;Ubicacion del Foco</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>&amp;Distancia</string>
        </property>
        <property name="buddy" >
         <cstring>sbDistance</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="sbDistance" >
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>999999.989999999990687</double>
        </property>
        <property name="singleStep" >
         <double>10.000000000000000</double>
        </property>
        <property name="value" >
         <double>250.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Factor de Intensidad</string>
        </property>
        <property name="buddy" >
         <cstring>sbFpeak</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>&amp;Profundidad</string>
        </property>
        <property name="buddy" >
         <cstring>sbDepth</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="sbDepth" >
        <property name="decimals" >
         <number>2</number>
        </property>
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>10.000000000000000</double>
        </property>
        <property name="value" >
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="sbSstrength" >
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Parametros</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Frecuencia Pico </string>
        </property>
        <property name="buddy" >
         <cstring>sbFpeak</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbFpeak" />
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Frecuencia Maxima</string>
        </property>
        <property name="buddy" >
         <cstring>sbFmax</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="sbFmax" />
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>&amp;Opciones</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Tiempo de &amp;Simulacion (segs)</string>
        </property>
        <property name="buddy" >
         <cstring>sbTMax</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="sbTMax" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Titulo</string>
        </property>
        <property name="buddy" >
         <cstring>leTitle</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="leTitle" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>265</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="chkLoop" >
     <property name="text" >
      <string>&amp;Repetir Simulacion</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
